CC/Sounder/files/renew-conf-two-client-3.6g.json

```diff
@@ -25,7 +25,7 @@
 
         "Clients" : {
                  "sdr_id" : [
-                     "RF3E000029"
+                     "RF3E000080"
```

Comment:
This is probably unnecessary to check in.

Reply:
yeah this shouldn't be there